TicketBookingSystemCore/Controllers/OrderController.cs

```
@@ -15,7 +15,7 @@ public async Task<IActionResult> GetCartItems(Guid cart_id)
         {
             var seats = await _orderService.GetCartItems(cart_id);
 
-            if (seats == null)
+            if (seats == null || !seats.Any())
             {
                 return NotFound();
             }
```

This validation looks odd - you check seats after getting them from service. In this case - `Get By CartId` we should rather fail inside service with bad data and throw an exception if CartId or items for CartId are missing or other validations failed, and then handle particular exceptions in ExceptionFilter returning the appropriate response status code.

---

Removed validations from controllers